gui/wxpython/gui_core/forms.py

```diff
@@ -2766,10 +2765,9 @@ def OnVerbosity(self, event):
                 if quiet.IsChecked():
                     quiet.SetValue(False)
                     self.task.get_flag("quiet")["value"] = False
-            else:
-                if verbose.IsChecked():
-                    verbose.SetValue(False)
-                    self.task.get_flag("verbose")["value"] = False
+            elif verbose.IsChecked():
```

Comment:
Lines 2765/2764 to 2768/2767 could be reduced to:
```
            if event.GetId() == verbose.GetId() and quiet.IsChecked():
                quiet.SetValue(False)
                self.task.get_flag("quiet")["value"] = False
```

Reply:
To apply

Reply:
Done